app/javascript/dashboard/routes/dashboard/settings/teams/Edit/Index.vue

```diff
@@ -0,0 +1,34 @@
+<template>
```

Comment:
Why do we have the same template and methods for both edit and add. Can't it be done with the same component?

Reply:
Would make the component ugly with checks for edit/new and translations